resources/aws/bucket.go

```diff
@@ -50,3 +50,42 @@ func (b *Bucket) Delete() error {
 
 	return nil
 }
+
+type BucketObject struct {
+	Name   string
+	Data   string
+	Bucket *Bucket
+	AWSEntity
+}
+
+func (bo *BucketObject) CreateIfNotExists() (bool, error) {
+	return false, notImplementedMethodError
```

Comment:
OK, now I don't understand why you don't use `MaskAny` here, and you use it below.. :confused: